app/lib/shared/trace_context.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
```

Comment:
If we don't expect that this will grow further, the code here could go into `shared/handlers.dart`.

Reply:
I looked around for a good spot for it. I'd have to add an import to `shared/handlers.dart` to every entry point, too.

I kinda like keeping it self-contained.

Although centralizing error handling, etc could be interesting in the future